students/aravichander/lesson09/test_mailroom.py

```diff
@@ -0,0 +1,34 @@
+
+import unittest
+from Mailroom_class import Donor, Donor_db
+
+class MyFuncTestCase(unittest.TestCase):
+	def test_donor_class(self):
```

Comment:
this is putting way too much in a single test.

each method / property should get its own test.

when they all pass, this is fine, but if one fails, the rest of the tests won't be run -- and you want all of them to run every time.